src/pages/Book/index.js

```diff
@@ -29,7 +30,32 @@ class Book extends React.Component {
     this.getBook(isbn);
   }
 
+  handleClick = (id) => {
+    const book = { ...this.state.data };
+    book.favorited = !book.favorited;
+    this.setState({ data: book });
+    this.props.handleClick(id);
+  };
+
   render() {
+    let button = "";
+    if (
+      this.props.favorites &&
+      this.state.data &&
+      this.props.favorites.includes(this.state.data.id)
+    ) {
+      button = (
+        <Favorite onClick={() => this.handleClick(this.state.data.id)}>
+          ★
+        </Favorite>
+      );
+    } else {
+      button = (
+        <Favorite onClick={() => this.handleClick(this.state.data.id)}>
+          ☆
+        </Favorite>
+      );
+    }
```

Comment:
why do you need this if else, they do the same thing

Reply:
I guess she wants to render a different icon depending on a condition, but I agree that this could be done more effectively